lib/ModelMixins/AsyncChartableMixin.ts

```diff
@@ -52,3 +52,13 @@ export default function AsyncChartableMixin<
 
   return AsyncChartableMixin;
 }
+
+namespace AsyncChartableMixin {
+  export interface AsyncChartableMixin
+    extends InstanceType<ReturnType<typeof AsyncChartableMixin>> {}
+  export function isMixedInto(model: any): model is AsyncChartableMixin {
+    return model && model.isChartable;
```

Comment:
I don't think this is a great check for AsyncChartableMixin.
The fact that `Chartable.is(model)` and `model.isChartable()` are not about the same thing is odd. I'm not sure `AsyncChartableMixin.isChartable` should exist. It might have been added to match UI existing logic, but we should prefer `Chartable.is(model)`.

I have no better suggestion though.

And I've just realised this comes straight from AsyncMappableMixin, where it was introduced under a `// TODO`: 
https://github.com/TerriaJS/terriajs/commit/5ea77ed4233a6a770f5e4352ee35edd69e8be1a6#diff-3ad2723f358849722e25021da0c6f044R22-R26

Reply:
Recorded in https://github.com/TerriaJS/terriajs/issues/4522

Reply:
The change is from another PR and is being addressed in this issue - https://github.com/TerriaJS/terriajs/issues/4522